python/mxnet/ndarray/numpy/_op.py

```diff
@@ -39,7 +39,7 @@
            'around', 'hypot', 'bitwise_xor', 'rad2deg', 'deg2rad', 'unique', 'lcm', 'tril', 'identity', 'take',
            'ldexp', 'vdot', 'inner', 'outer', 'equal', 'not_equal', 'greater', 'less', 'greater_equal', 'less_equal',
            'hsplit', 'rot90', 'einsum', 'true_divide', 'nonzero', 'shares_memory', 'may_share_memory', 'diff', 'resize',
-           'nan_to_num', 'where']
+           'nan_to_num', 'where', 'empty_like']
```

Comment:
move this operator to the position after `full` in the list and in the file, same for all python wrappers.